src/nvim/diff.c

```diff
@@ -2017,17 +2017,24 @@ int diff_infold(win_T *wp, linenr_T lnum)
 }
 
 /// "dp" and "do" commands.
-///
-/// @param put
-void nv_diffgetput(int put)
+void nv_diffgetput(bool put, size_t count)
 {
   exarg_T ea;
-  ea.arg = (char_u *)"";
+  char buf[30];
+
+  if (count == 0) {
+    ea.arg = (char_u *)"";
+  } else {
+    vim_snprintf(buf, 30, "%ld", count);
```

Comment:
I extracted inttypes.h from VS2015 latest preview and here it is: https://gist.github.com/jasonwilliams200OK/45d0b5f4fd2b087e0b3a

And here is the one that shipped with VS2013 Update 4: https://gist.github.com/jasonwilliams200OK/2a0ce733923fbcf86c64 (don't know why they say complex.h in comment in 2013 version?)

Besides the following, all C99 and C11 standard headers are included in CRT (VS2015):

```C
C11 - <stdalign.h>
C11 - <stdatomic.h>
C11 - <stdnoreturn.h>
C99 - <tgmath.h>
C11 - <threads.h>
```

The only C11 header that is included is [`<uchar.h>`](https://gist.github.com/jasonwilliams200OK/e8c8d4f71284d52ffa91), which means STL (person) was not kidding [in this blog's comment](http://blogs.msdn.com/b/vcblog/archive/2014/06/11/c-11-14-feature-tables-for-visual-studio-14-ctp1.aspx?PageIndex=1#comments):

> My coworker Pat Brenner is actually adding <uchar.h> to the CRT right now, and I've given him <cuchar> to add to the STL. Unless a meteor strikes, this will get into VS14 RTM.

(it was later, they changed the name to VS2015)

So now we even know what date uchar.h got included in their CRT! :)

BTW, that list in blog I linked above is now superseded by [this one](http://blogs.msdn.com/b/vcblog/archive/2014/11/17/c-11-14-17-features-in-vs-2015-preview.aspx).

PS Matter of time before we would be `@`'ing VC guys directly on GH..